common/static/common/js/components/collections/paging_collection.js

```diff
@@ -79,6 +79,9 @@
              * underlying server API.
              */
             getPage: function () {
+                // TODO: this.currentPage is currently returning a function sometimes when it is called.
```

Comment:
@dan-f I will provide more information about this. But I really don't think there is any change in behavior here, and we know that we will get rid of this code in a future story.